Build fails if not using at least one lambda function

npm build, when not having at least one lambda function, will fail with:

```
WebpackOptionsValidationError: Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
```

due to:
```
npm ERR! atlas@ build:functions: `netlify-lambda build assets/lambda`
npm ERR! Exit status 1
```
Issue gets fixed if I add at least one `.js` file with a dummy lambda function (eg. the one in the readme)

Full stack trace
```
netlify-lambda: Building functions

      ---Start netlify-lambda notification---
      WARNING: No valid single functions files (ending in .mjs, .js or .ts) were found. 
      This could be because you have nested them in a folder.
      If this is expected (e.g. you have a zipped function built somewhere else), you may ignore this.
      ---End netlify-lambda notification---
      
WebpackOptionsValidationError: Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.entry should be an non-empty object.
   -> Multiple entry bundles are created. The key is the chunk name. The value can be a string or an array.
    at webpack (/Users/sergio/Development/atlas/node_modules/webpack/lib/webpack.js:31:9)
    at /Users/sergio/Development/atlas/node_modules/netlify-lambda/lib/build.js:145:5
    at new Promise (<anonymous>)
    at Object.exports.run (/Users/sergio/Development/atlas/node_modules/netlify-lambda/lib/build.js:144:10)
    at Command.<anonymous> (/Users/sergio/Development/atlas/node_modules/netlify-lambda/bin/cmd.js:80:8)
    at Command.listener (/Users/sergio/Development/atlas/node_modules/commander/index.js:315:8)
    at Command.emit (events.js:210:5)
    at Command.parseArgs (/Users/sergio/Development/atlas/node_modules/commander/index.js:651:12)
    at Command.parse (/Users/sergio/Development/atlas/node_modules/commander/index.js:474:21)
    at Object.<anonymous> (/Users/sergio/Development/atlas/node_modules/netlify-lambda/bin/cmd.js:100:9)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! atlas@ build:functions: `netlify-lambda build assets/lambda`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the atlas@ build:functions script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/sergio/.npm/_logs/2020-03-25T20_59_32_319Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! atlas@ build: `hugo && npm run build:functions`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the atlas@ build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/assets/lambda/hello.js
+++ b/assets/lambda/hello.js
@@ -1,0 +1,11 @@
+exports.handler = (event, context, callback) => {
+    callback(null, {
+        statusCode: 200,
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+            message: "Hi from Lambda."
+        })
+    });
+}

